CHANGELOG.md

```diff
@@ -7,13 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.9.0 2024-05-02
```

Comment:
This is our first real breaking change for something that is in production. Here's what I think we should do:

1.     Keep EIN in the spec and response sets, but mark it as deprecated.
2.     Release the new version (and new sdk) which has both.
3.     Update the front end to use taxId
4.     Open a new PR that removes the deprecated ein.

Reply:
This PR now reflects this change in approach -- Organizations have both Tax ID and EIN right now; EIN is marked as deprecated.

Once we release this, the new SDK will have both, we can migrate our client.

In a future world, this is the approach we would take and then we'd plan to remove the deprecated field at a later date; for now we know we're the only client so we can act more quickly and remove the old value sooner.